src/docker/files/ContainerFilesUtils.ts

```diff
@@ -221,161 +221,39 @@ export async function statLinuxContainerItem(executor: DockerContainerExecutor,
-    let parsedObject: { [key: string]: string };
-
-    for (const line of lines) {
-        const index = line.indexOf('=');
-
-        if (index > 0) {
-            const name = line.substr(0, index);
-            const value = line.substr(index + 1);
-
-            if (parsedObject === undefined) {
-                parsedObject = {};
-            }
-
-            parsedObject[name] = value;
-        }
-    }
-
-    return parsedObject;
-}
-
-function parseWmiTime(wmiTime: string): number | undefined {
-    if (wmiTime) {
-        const match = /^(?<year>\d{4})(?<month>\d{2})(?<day>\d{2})(?<hour>\d{2})(?<minute>\d{2})(?<second>\d{2})\.(?<micro>\d{6})(?<offset>[+-]\d{3})$/.exec(wmiTime);
-
-        if (match) {
-
-            const options = {
-                year: parseInt(match.groups.year, 10),
-                month: parseInt(match.groups.month, 10) - 1,
-                day: parseInt(match.groups.day, 10),
-                hour: parseInt(match.groups.hour, 10),
-                minute: parseInt(match.groups.minute, 10),
-                second: parseInt(match.groups.second, 10),
-                millisecond: parseInt(match.groups.micro, 10) / 1000
-            };
-
-            // TODO: Add ObjectSupport constructor to type definitions.
-            const time = dayjs(<dayjs.ConfigType><unknown>options).utcOffset(parseInt(match.groups.offset, 10));
-
-            return time.valueOf();
-        }
-    }
-
-    return undefined;
-}
-
-const CreationDate = 'CreationDate';
-const FileSize = 'FileSize';
-const LastModified = 'LastModified';
-
-async function statWindowsContainerDirectory(executor: DockerContainerExecutor, itemPath: string): Promise<DirectoryItemStat | undefined> {
-    if (/^[a-zA-Z]:\\$/.test(itemPath)) {
-
-        //
-        // For root directories, assume they exist and return a faked stat...
-        //
-        // TODO: Find a WMI command that provides such properties for root directories.
-        //
-
-        return {
-            ctime: 0,
-            mtime: Date.now(),
-            size: 0,
-            type: 'directory'
-        };
-    }
-
-    const parsedPath = path.win32.parse(itemPath);
-
-    const drive = parsedPath.root.replace(/\\/, '');
-    const wmipath = parsedPath.dir.concat('\\');
-    const filename = parsedPath.base;
-    const command = ['cmd', '/C', `wmic fsdir where "drive='${drive}' and path='${wmipath}' and filename='${filename}'" get ${CreationDate}, ${LastModified} /format:list`];
+export async function statWindowsContainerItem(executor: DockerContainerExecutor, itemPath: string, itemType: DirectoryItemType): Promise<DirectoryItemStat | undefined> {
+    const command: string[] = ['powershell', '-Command', `$finfo = Get-Item -Path '${itemPath}'; Write-Output ('{0};{1};{2};{3}' -f ([System.DateTimeOffset]$finfo.CreationTimeUtc).ToUnixTimeMilliseconds(), ([System.DateTimeOffset]$finfo.LastWriteTimeUtc).ToUnixTimeMilliseconds(), $finfo.Length, @('file','directory')[$finfo.PSIsContainer])`];
```

Comment:
PowerShell 5 is present in containers I checked (20H2, 1809, Nanoserver LTSC2022) and lacks a ternary operator, so I used the insane-but-functional workaround suggested by many users online of creating a two-element array and using the true/false result to index into it:
```powershell
@('file','directory')[$finfo.PSIsContainer]
```